pkg/k8s/client.go

```diff
@@ -469,35 +475,35 @@ func UpdateCNPStatus(cnpClient CNPCliInterface, timeout time.Duration,
 				break
 			}
 			if err != nil {
-				log.Warningf("k8s: unable to get k8s CNP %s/%s from local cache: %s", ns, name, err)
+				scopedLog.WithError(err).Warn("unable to get k8s CNP from local cache")
 				break
 			}
 			serverRule, ok := serverRuleStore.(*CiliumNetworkPolicy)
 			if !ok {
-				log.Warningf("Received unknown object %+v, expected a CiliumNetworkPolicy object", serverRuleStore)
+				scopedLog.WithError(err).WithField(logfields.Object, serverRuleStore).Warn("Received unknown object, expected a CiliumNetworkPolicy object")
```

Comment:
`"Received object of unknown type from API server, expecting CiliumNetworkPolicy"`